thoth/storages/graph/models.py

```diff
@@ -1688,6 +1687,29 @@ class SIAggregated(Base, BaseExtension):
     python_package_version_entity = relationship("PythonPackageVersionEntity", back_populates="si_aggregated_runs")
 
 
+class ImportPackage(Base, BaseExtension):
+    """Packages imported as a result of solver run"""
+
+    __tablename__ = "import_package"
+
+    id = Column(Integer, primary_key=True, autoincrement=True, nullable=False)
+    import_package_name = Column(Text, nullable=True)
+
+    python_package_versions = relationship("FoundImportPackage", back_populates="import_package")
+
```

Comment:
I actually don't see the index created for `import_package_name` as @fridex suggested. It's missing the migration also indeed, there should be something like `op.create_index`

Reply:
> I actually don't see the index created for `import_package_name` as @fridex suggested. It's missing the migration also indeed, there should be something like `op.create_index`

I understood as `id` field is index in line 1695:
`id = Column(Integer, primary_key=True, autoincrement=True, nullable=False)`

![Screen Shot 2021-08-12 at 5 33 48 PM](https://user-images.githubusercontent.com/58737562/129273057-9b097aa5-fc07-4400-bc9f-2ae253efc6fc.png)

Or if it is not that i can definitely work on that.

Reply:
See for example: https://github.com/thoth-station/storages/blob/61b9de2fa41a31547bb1b45ce7b21276455e70e7/thoth/storages/graph/models.py#L215. I think @fridex meant to have one specific for that, the ID has an index by default I think.

Reply:
Now clear. Implemented in d8d35af04d26d99a2f8187c78f267a6639e026a6
And created new alembic migration with adjustments to it that you pointed out previously

Reply:
> See for example:
> 
> https://github.com/thoth-station/storages/blob/61b9de2fa41a31547bb1b45ce7b21276455e70e7/thoth/storages/graph/models.py#L215
> . I think @fridex meant to have one specific for that, the ID has an index by default I think.

Now clear. Implemented in d8d35af04d26d99a2f8187c78f267a6639e026a6
And created new alembic migration with adjustments to it that you pointed out previously
Let me know if there is anything else)